SQL/0000-00-00-schema.sql

```diff
@@ -1643,6 +1671,386 @@ CREATE TABLE `parameter_type_override` (
+  `SpecimenTypeID` integer unsigned NOT NULL,                                   
+  `Quantity` DECIMAL(10, 3) NOT NULL,                                           
+  `UnitID` integer unsigned NOT NULL,                                           
+  `SessionID` integer unsigned UNSIGNED NOT NULL,                               
+  CONSTRAINT `PK_biobank_specimen` PRIMARY KEY (`SpecimenID`),                  
+  CONSTRAINT `FK_biobank_specimen_ContainerID`                                  
+    FOREIGN KEY (`ContainerID`) REFERENCES `biobank_container`(`ContainerID`)   
+    ON UPDATE RESTRICT ON DELETE RESTRICT,                                      
+  CONSTRAINT `FK_biobank_specimen_SpecimenTypeID`                               
+    FOREIGN KEY (`SpecimenTypeID`) REFERENCES `biobank_specimen_type`(`SpecimenTypeID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT,                                      
+  CONSTRAINT `FK_biobank_specimen_UnitID`                                       
+    FOREIGN KEY (`UnitID`) REFERENCES `biobank_unit` (`UnitID`)                 
+    ON UPDATE RESTRICT ON DELETE RESTRICT,                                      
+  CONSTRAINT `FK_biobank_specimen_SessionID`                                    
+    FOREIGN KEY (`SessionID`) REFERENCES `session`(`ID`)                        
+    ON UPDATE RESTRICT ON DELETE RESTRICT,                                      
+  CONSTRAINT `UK_biobank_specimen_ContainerID` UNIQUE (`ContainerID`)           
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_specimen_freezethaw` (                                    
+  `SpecimenID` integer unsigned NOT NULL,                                       
+  `FreezeThawCycle` integer unsigned NOT NULL,                                  
+  CONSTRAINT `PK_biobank_specimen_freezethaw` PRIMARY KEY (`SpecimenID`),       
+  CONSTRAINT `FK_biobank_specimen_freezethaw_SpecimenID`                        
+    FOREIGN KEY (`SpecimenID`) REFERENCES `biobank_specimen`(`SpecimenID`)      
+    ON UPDATE RESTRICT ON DELETE RESTRICT                                       
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_specimen_collection` (                                    
+  `SpecimenID` integer unsigned NOT NULL,                                       
+  `Quantity` DECIMAL(10, 3) NOT NULL,                                           
+  `UnitID` integer unsigned NOT NULL,                                           
+  `CenterID` integer unsigned NOT NULL,                                         
+  `Date` DATE NOT NULL,                                                         
+  `Time` TIME NOT NULL,                                                         
```

Comment:
Why not use DATETIME? Or TIMESTAMP?

Reply:
Because the `date` and `time` fields are separate I figure d it would be better to have them be the datatypes of the data they would respectively hold. I don't think it should be a timestamp because this data is inputted client-side, but if you think `DateTime` would be better, I could change it to that.

https://dev.mysql.com/doc/refman/8.0/en/datetime.html

Reply:
timestamp relies on server settings for timezone and if it is user inputed, it might lead to error. Datetime seems more fitting.

Reply:
Right, but I'm not using timestamp. What's the issue with simply using `Date` and `Time` if not storing `DateTime`s?